### PR TITLE
[953] Set provider.changed_at

### PR DESCRIPTION
**Pull request by davidgisbey**
Will be used to return changed records in the UCAS API

### CODE REVIEW

File: app/models/provider.rb [hunk 1]
```diff
@@ -60,4 +62,10 @@ def address_info
       .attributes_before_type_cast
       .slice('address1', 'address2', 'address3', 'address4', 'postcode', 'region_code')
   end
+
+private
+
+  def update_changed_at
+    self.changed_at = Time.now
```
Comment (dankmitchell) [inline app/models/provider.rb]: `Time.now.utc` ?

File: spec/models/provider_spec.rb [hunk 2]
```diff
@@ -237,4 +237,16 @@
...
   end
+
+  describe '#changed_at' do
+    context 'record is modified' do
+      let!(:provider) { create(:provider, changed_at: 1.hour.ago) }
+
+      before { provider.contact_name = "bob" }
+
+      it 'should have been updated'do
+        expect(provider.changed_at).to be_within(1.second).of(Time.now)
```
Comment (defong) [inline spec/models/provider_spec.rb]: `Time.now.utc` ?

Comment (davidgisbey) [reply spec/models/provider_spec.rb]: Thanks! Will change

File: app/models/provider.rb (hunk 1 above)
Comment (davidgisbey) [reply app/models/provider.rb]: Thanks! Will change

**Comment by misaka** [comment]
What do we think about patching into `timestamp_attributes_for_update` like this to update `changed_at` automagically?

**Pull request closed without merging**